README.md

```diff
@@ -1 +1,88 @@
-# wine_service
+# Weather Vine Wine Microservice
+
+## About this Project
+Weather Vine is an educational app for consumers to connect more deeply with the wine they enjoy. Explore wines from a region and see how the climate has influenced the very wine one drinks!   
+
+## Table of Contents
+
+  - [Getting Started](#getting-started)
+  - [Running the tests](#running-the-tests)
+  - [Service Oriented Architecture](#service-oriented-architecture)
+  - [Other Repos](#other-repos)
+  - [Database Schema](#database-schema)
+  - [Built With](#built-with)
+  - [Contributing](#contributing)
+  - [Versioning](#versioning)
+  - [Authors](#authors)
+  - [License](#license)
+  - [Acknowledgments](#acknowledgments)
+
+## Getting Started
+
+To get the web application running, please fork and clone down the repo.
+`git clone <your@github.account:WeatherVine/front_end.git>`
+
+### Prerequisites
+
+To run this application you will need Ruby 2.5.3 and Rails 5.2.5
+
+### Installing
+
+- Install the gem packages  
+`bundle install`
+
+- Create the database by running the following command in your terminal
+`rails db:{drop,create,migrate}`
+
+## Running the tests
+RSpec testing suite is utilized for testing this application.
+- Run the RSpec suite to ensure everything is passing as expected  
+`bundle exec rspec`
+
+## Service Oriented Architecture
+![](assets/README-f315b6d0.png)
+
+## Other Repos
+- Weather Vine Front-End Application
+  - [Repo](https://github.com/WeatherVine/front_end) | [Heroku](https://weathervine-fe.herokuapp.com/)
+- Weather Vine Back-End Application
+  - [Repo](https://github.com/WeatherVine/back_end) | [Heroku](https://weathervine-be.herokuapp.com/)
+- Weather Vine Weather Microservice
+  - [Repo](https://github.com/WeatherVine/weather_service) | [Heroku](https://weather-service-sinatra.herokuapp.com/)
```

Comment:
I like this section! It keeps everything connected. Good add! I will probably take it for the FE and BE READMEs.